x-pack/plugin/src/test/resources/rest-api-spec/api/data_frame.get_data_frame_transform.json

```diff
@@ -4,12 +4,29 @@
     "methods": [ "GET" ],
     "url": {
       "path": "/_data_frame/transforms/{transform_id}",
-      "paths": [ "/_data_frame/transforms/{transform_id}"],
+      "paths": [ "/_data_frame/transforms/{transform_id}", "/_data_frame/transforms"],
       "parts": {
         "transform_id": {
           "type": "string",
           "required": false,
-          "description": "The id of the transforms to get, '_all' or '*' implies get all transforms"
+          "description": "The id or comma delimited list of id expressions of the transforms to get, '_all' or '*' implies get all transforms"
+        }
+      },
+      "params": {
+        "from": {
+          "type": "int",
+          "required": false,
+          "description": "skips a number of transform configs, defaults to 0"
+        },
+        "size": {
+          "type": "int",
+          "required": false,
+          "description": "specifies a max number of transforms to get, defaults to 100"
+        },
+        "allow_no_transforms": {
+          "type": "boolean",
+          "required": false,
+          "description": "Whether to ignore if a wildcard expression matches no transforms. Defaults to true (This includes `_all` string or when no jobs have been specified)"
```

Comment:
Can you explain the usecase for `allow_no_transforms=false` ? 

I think we should not introduce additional parameters if we do not have an immediate (non-future), well defined usage.

Reply:
Somebody querying a known transform prefix `transform-type*` and wanting to have an error if none are found instead of an empty array.

Reply:
he can check `count`? The search API doesn't provide something like that, if you do a `matches_nothing*/_search` you will get a valid reply and as far as I know there is nothing comparable to `allow_no_transforms`

Reply:
I will remove this parameter as there are no users clamoring for it :). We can easily add it back again in the future.